pkg/extract/types/data/detection/condition/criteria/criterion/versioncriterion/versioncriterion.go

```diff
@@ -150,6 +150,13 @@ func (c Criterion) Accept(query Query) (bool, error) {
 		if c.Affected == nil {
 			return true, nil
 		}
+
+		// Workaround for RHEL oval v2 -> vex transition. Ideally, query.Source.Version MUST NOT be empty.
+		// This MUST BE REMOVED soon.
```

Comment:
In RHEL, the Source Package is only required for unfixed vulnerabilities, in which case Affected should always be nil. If Affected is not nil, it expects to compare Versions, so it should be compared even if the query version is empty.

Reply:
OK. No change needed!